setup.py

```diff
@@ -156,25 +159,26 @@
         else:
             debug_flag = False
         copy_args.remove(x)
-    
+
 if not incdirs:
     raise Exception("FMI Library cannot be found. Please specify its location, either using the flag to the setup script '--fmil-home' or specify it using the environment variable FMIL_HOME.")
 
-#Check to see if FMILIB_SHARED exists and if so copy it
 if 0 != sys.argv[1].find("clean"): #Dont check if we are cleaning!
+
+    #Check to see if FMILIB_SHARED exists and if so copy it, otherwise raise exception
     if sys.platform.startswith("win"):
-        try:
-            files = os.listdir(os.path.join(libdirs))
-        except Exception:
-            raise Exception("The FMI Library binary cannot be found at path: "+str(os.path.join(libdirs)))
-        for file in files:
-            if "fmilib_shared" in file and not file.endswith("a"):
-                shutil.copy2(os.path.join(libdirs,file),os.path.join(".","src","pyfmi"))
-                fmilib_shared = os.path.join(".","src","pyfmi",file)
-                break
-        else:
-            raise Exception("Could not find FMILibrary at: %s"%libdirs)
-            
+        dirs_to_search = [libdirs, bindirs]
```

Comment:
I removed the try/catch. Both `os.listdir` and `shutil.copy2` can raise a `FileNotFoundError` but I think that the message was already descriptive enough and prefer just leaving this to Python.